experimental/traffic-portal/build/traffic-portal.spec

```diff
@@ -0,0 +1,96 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# RPM spec file for the Traffic Portal
+#
+%define   debug_package %{nil}
+Name:     traffic-portal
+Version:  %{traffic_control_version}
+Release:  %{build_number}
+Summary:  Traffic Portal v2
+Group:    Applications/Communications
+License:  Apache License, Version 2.0
+URL:      https://github.com/apache/trafficcontrol/
+Source:   %{_sourcedir}/traffic-portal-%{traffic_control_version}.tgz
+AutoReqProv: no
+Requires: nodejs >= 2:16.0.0
+Requires(pre): /usr/sbin/useradd, /usr/bin/getent
+
+%define traffic_portal_home /opt/traffic-portal
+%define traffic_portal_log /var/log/traffic-portal
+%define traffic_portal_conf /etc/traffic-portal
+%define traffic_portal traffic-portal-%{version}
+%define traffic_portal_user trafficportal
+%description
+Installs Traffic Portal
+
+Built: @BUILT@
+
+%prep
+%__rm -rf $RPM_BUILD_DIR/%{traffic_portal}
+tar -xzvf $RPM_SOURCE_DIR/%{traffic_portal}.tgz
+
+%pre
+/usr/bin/getent group %{traffic_portal_user} || /usr/sbin/groupadd -r %{traffic_portal_user}
+/usr/bin/getent passwd %{traffic_portal_user} || /usr/sbin/useradd -r -d %{traffic_portal_home} -s /sbin/nologin %{traffic_portal_user} -g %{traffic_portal_user}
+
+%build
+cd ${RPM_BUILD_DIR}/%{traffic_portal}
+npm run build:ssr
+
+%install
+%__mkdir -p ${RPM_BUILD_ROOT}/etc/init.d
+%__mkdir -p ${RPM_BUILD_ROOT}/etc/logrotate.d
+%__mkdir -p ${RPM_BUILD_ROOT}%{traffic_portal_conf}
+%__mkdir -p ${RPM_BUILD_ROOT}%{traffic_portal_home}/browser
+%__mkdir -p ${RPM_BUILD_ROOT}%{traffic_portal_home}/server
+%__mkdir -p ${RPM_BUILD_ROOT}%{traffic_portal_home}/node_modules
+%__mkdir -p ${RPM_BUILD_ROOT}%{traffic_portal_log}
+
+%__cp ${RPM_BUILD_DIR}/%{traffic_portal}/build/config.json ${RPM_BUILD_ROOT}%{traffic_portal_conf}/.
+%__cp -r ${RPM_BUILD_DIR}/%{traffic_portal}/dist/traffic-portal/* ${RPM_BUILD_ROOT}%{traffic_portal_home}/.
+%__cp -r ${RPM_BUILD_DIR}/%{traffic_portal}/build/node_modules ${RPM_BUILD_ROOT}%{traffic_portal_home}/.
+%__cp ${RPM_BUILD_DIR}/%{traffic_portal}/build/etc/init.d/traffic-portal ${RPM_BUILD_ROOT}/etc/init.d/.
+%__cp ${RPM_BUILD_DIR}/%{traffic_portal}/build/etc/logrotate.d/traffic-portal ${RPM_BUILD_ROOT}/etc/logrotate.d/.
+%__cp ${RPM_BUILD_DIR}/%{traffic_portal}/LICENSE ${RPM_BUILD_DIR}/.
+
+# creates dynamic json file needed at runtime for traffic portal to display release info
+echo "{
+	\"date\": \"$(date +'%Y-%m-%d %H:%M')\",
+	\"elRelease\": \"%{rhel_vers}\",
+	\"hash\": \"%{build_number}\",
+	\"version\": \"%{version}\"
+}" > ${RPM_BUILD_ROOT}%{traffic_portal_conf}/version.json
+
+%post
+%__chmod +x %{traffic_portal_home}/node_modules/pm2/bin/pm2
+echo "Successfully installed traffic-portal to " %{traffic_portal_home}
+/sbin/chkconfig traffic-portal on
+echo ""
+echo "Start with 'systemctl start traffic-portal' or by running '%{traffic_portal_conf}/traffic-portal'"
+
```

Comment:
When installing the RPM with 

```shell
rpm -e traffic-portal
```
, a `/opt/traffic-portal` still exists (but is empty). That should be removed too, right?